gems/sorbet-runtime/lib/types/types/typed_range.rb

```diff
@@ -15,10 +15,15 @@ def name
     end
 
     # @override Base
-    def valid?(obj)
+    def recurisvely_valid?(obj)
```

Comment:
Typo?

Reply:
Oof, yes. …apparently not only do we not test that, we must not use it anywhere in pay-server, either.